lib/gettext.js

```diff
@@ -8,8 +8,7 @@
     var defaults = {
       domain: 'messages',
       locale: document.documentElement.getAttribute('lang') || 'en',
-      plural_form: 'nplurals=2; plural=(n!=1);',
-      plural_func: function (n) { return { nplurals: 2, plural: (n!=1) ? 1 : 0 }; },
+      plural_func: function (n) { return { nplural: 2, plural: (n!=1) ? 1 : 0 }; },
```

Comment:
Ok I get it. Though, it might be better to keep `nplurals` here and modify [the eval function here](https://github.com/guillaumepotier/gettext.js/blob/master/lib/gettext.js#L59) to return `nplurals` and in [this condition](https://github.com/guillaumepotier/gettext.js/blob/master/lib/gettext.js#L86) use `plural.plural > plural.nplurals`.

It would be more coherent. WDYT?

Best